library/build.gradle

```diff
@@ -120,9 +120,9 @@ dependencies { configuration ->
     implementation mapboxAnnotationPlugin
 
     // Comment the line below when creating releases - The line is for development of the library & utils
-    //implementation (project(":utils")) {
+    implementation (project(":utils")) {
     // Uncomment the line below when creating releases
-    implementation('io.ona.kujaku:utils:0.10.6-SNAPSHOT') {
+    //implementation('io.ona.kujaku:utils:0.10.7-SNAPSHOT') {
```

Comment:
I think we should revert this change and only have it like this during development. That way, the codebase will always be ready for published snapshots.

Reply:
We can fix later with some conditional logic for better maintainability.